keps/sig-node/4569-cgroup-v1-deprecation/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: Deprecate Support for cgroup v1 in Kubernetes
+kep-number: 4569
+authors:
+  - "@harche"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: implementable
+creation-date: 2024-04-04
+reviewers:
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
+approvers:
+  - "@derekwaynecarr"
+  - "@dchen1107"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
```

Comment:
We're planning to deprecate a thing, right? So I imagine this would go in as deprecated.